keyboards/keebsforall/freebird60/readme.md

```diff
@@ -0,0 +1,19 @@
+# Freebird60
+
+![freebird60](https://imgur.com/a/cfYz93b)
```

Comment:
```suggestion
![freebird60](https://i.imgur.com/qGtVVbyl.jpeg)
```
Needs to be a direct link.

Reply:
Apologies. I had pasted that in to the file and didn't save before committing. LGTM